.travis.yml

```diff
@@ -1,16 +1,11 @@
 language: ruby
 sudo: false
 rvm:
-  - 2.3.4
   - 2.4.1
 gemfile:
-  - gemfiles/rails_4.2.gemfile
   - gemfiles/rails_5.gemfile
-services:
-  - redis-server
```

Comment:
redis is core service to be used in production, we need to be sure, lit work on redis properly